buzsaki_lab_to_nwb/original-code/new_convert_yuta.py

```diff
@@ -33,19 +37,26 @@
             'genotype': 'POMC-Cre::Arch',
             'species': 'mouse' # should be Mus musculus?
     },
-    'neuroscope': {
-        'Device': {
-            'name': 'implant',
-            'description': 'no description'
-        },
-        'ElectrodeGroup': {
-            'name': 'electrodes',
-            'description': 'no description'
-        },        
-        'ElectricalSeries': {
-            'name': 'ElectricalSeries',
-            'description': 'no description'
+    'NeuroscopeRecording': {
+        'Ecephys': {
+            # NwbRecordingExtractor expects metadata to be lists of dictionaries
+            'Device': [{
+                'name': 'implant',
+                'description': 'YutaMouse41-150903.xml'
+            }],
+            'ElectrodeGroup': [{
+                'name': f'shank{n+1}',
+                'description': f'shank{n+1} electrodes',
+                'location': 'unknown',
+                'device': 'implant'
+            } for n,a in enumerate(range(8))] # We know there are 8 shanks, but how best to integrate an auto-detect in here? i.e., at what point do we incorporate the xml info?
```

Comment:
This obviously shouldn't be hard coded, but it remains to be decided where and how the .XML parameters should be retrieved and stored in the previous steps. Perhaps added to the __init__ of the Buzsaki specific converter?